src/typescript/frontend/src/components/header/wallet-button/ConnectWalletButton.tsx

```diff
@@ -67,41 +69,82 @@ export const ButtonWithConnectWalletFallback: React.FC<ConnectWalletProps> = ({
-              style={{ width, maxWidth: width }}
-              ref={ref}
-            />
-            {connected ? (
-              <p className="text-base flex mt-1.5 animate-flicker drop-shadow-voltage">{"⚡"}</p>
-            ) : (
-              <div className="pl-2.5">{"}"}</div>
-            )}
-          </div>
-        </Button>
-      ) : (
-        children
-      )}
-    </>
-  );
+  if (!mobile) {
+    return (
+      <>
+        {!connected || !children ? (
+          <Button
+            className={className}
+            onClick={(e) => {
+              e.preventDefault();
+              onClick ? onClick() : connectWallet();
+              handleReplay();
+            }}
+            onMouseOver={handleReplay}
+          >
+            <div className="flex flex-row text-ec-blue text-2xl">
+              {connected ? (
+                <p className="text-base flex mt-1.5 animate-flicker drop-shadow-voltage">{"⚡"}</p>
+              ) : (
+                <div className="pr-2.5">{"{"}</div>
+              )}
+              <p
+                className="uppercase whitespace-nowrap text-overflow-ellipsis overflow-hidden"
+                style={{ width, maxWidth: width }}
+                ref={ref}
+              />
+              {connected ? (
+                <p className="text-base flex mt-1.5 animate-flicker drop-shadow-voltage">{"⚡"}</p>
+              ) : (
+                <div className="pl-2.5">{"}"}</div>
+              )}
+            </div>
+          </Button>
+        ) : (
+          children
+        )}
+      </>
+    );
+  } else {
+    return (
+      <>
+        {!connected || !children ? (
+          <StyledItemWrapper onMouseOver={replay} onClick={onClick} borderBottom={true}>
+            <Button
+              className={className}
+              onClick={(e) => {
+                e.preventDefault();
+                onClick ? onClick() : connectWallet();
+                handleReplay();
+              }}
+              onMouseOver={handleReplay}
+            >
+              <div className="flex flex-row pixel-heading-3">
+                {connected && (
+                  <p className="text-base flex mt-1.5 animate-flicker drop-shadow-voltage">
+                    {"⚡"}
+                  </p>
+                )}
+                <p
+                  className="uppercase whitespace-nowrap text-overflow-ellipsis overflow-hidden"
+                  style={{ width, maxWidth: width }}
+                  ref={ref}
+                />
+                {connected && (
+                  <p className="text-base flex mt-1.5 animate-flicker drop-shadow-voltage">
+                    {"⚡"}
+                  </p>
+                )}
+              </div>
+            </Button>
+            <Arrow width="18px" color="black" />
+          </StyledItemWrapper>
+        ) : (
+          children
+        )}
+      </>
+    );
+  }
```

Comment:
Instead of duplicating the component I think we can just do something like this:

https://github.com/econia-labs/emojicoin-dot-fun/blob/891f958cd4fa13d4cb5661eab22ede3466aac060/src/typescript/frontend/src/components/header/wallet-button/ConnectWalletButton.tsx#L62

Especially since after my merge a cople of things change here- I changed the voltage `p` elements to use `mt-2.5` for example so they're centered better

You mind if we just use the other code here? that is, the re-ordering so connect wallet is first on mobile?

Reply:
Here's the other part to that:

https://github.com/econia-labs/emojicoin-dot-fun/blob/4990c9f8bdbf04a354567b580397f55e62453653/src/typescript/frontend/src/components/header/wallet-button/ConnectWalletButton.tsx#L75